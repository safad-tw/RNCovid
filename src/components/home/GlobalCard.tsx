import React, { memo } from 'react';
import {
  Dimensions
} from 'react-native';
import {
  BarChart
} from 'react-native-chart-kit'

import LanguagesStrings from './../../localization/LanguagesStrings';
import {GlobalData} from './../../models/GlobalData'


interface GlobalCardProps {
  data: GlobalData
}



const GlobalCard = (props: GlobalCardProps) => {

  const getBarChartData = () => {
    if (props.data) {
      return [props.data.TotalDeaths / 1000, props.data.TotalRecovered / 1000, (props.data.TotalConfirmed - props.data.TotalRecovered) / 1000]
    }
    return []
  }

  return (
    <>
      <BarChart
        data={{
          labels: [LanguagesStrings.globalDeaths, 
            LanguagesStrings.recoveries, 
            LanguagesStrings.activeCases],
           datasets: [
            {
              data: getBarChartData()
            },
          ],
        }}
        width={Dimensions.get('window').width - 16}
        height={250}
        yAxisLabel={''}
        xAxisLabel={'(K)'}
        yAxisSuffix={''}
        chartConfig={{
          backgroundColor: '#1cc910',
          backgroundGradientFrom: '#eff3ff',
          backgroundGradientTo: '#efefef',
          decimalPlaces: 0,
          color: (opacity = 1) => `rgba(0, 0, 0, ${opacity})`,
          style: {
            borderRadius: 16,
          },
        }}
        style={{
          marginVertical: 8,
          borderRadius: 16,
        }}
      />
    </>);

}


export default memo(GlobalCard);