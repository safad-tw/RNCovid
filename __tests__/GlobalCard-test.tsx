/**
 * @format
 */

import 'react-native';
// import App from '../App';

import React from 'react';
import GlobalCard from '../src/components/home/GlobalCard';
import renderer from 'react-test-renderer';



it('renders correctly', () => {
  let globalData = {
    NewConfirmed: 257853,
    TotalConfirmed: 232449101,
    NewDeaths: 6292,
    TotalDeaths: 4762406,
    NewRecovered: 0,
    TotalRecovered: 0,
    Date: "2021-09-29T12:04:59.613Z"
    }
  const tree = renderer.create(<GlobalCard data={globalData} />);
  expect(tree).not.toBeNull()
});

it('Renders snapshot as expected', () => {
  let globalData = {
    NewConfirmed: 257853,
    TotalConfirmed: 232449101,
    NewDeaths: 6292,
    TotalDeaths: 4762406,
    NewRecovered: 0,
    TotalRecovered: 0,
    Date: "2021-09-29T12:04:59.613Z"
    }
  const tree = renderer.create(<GlobalCard data={globalData} />).toJSON();
  expect(tree).toMatchSnapshot();
});


it('Find Graph element', () => {
  let globalData = {
    NewConfirmed: 257853,
    TotalConfirmed: 232449101,
    NewDeaths: 6292,
    TotalDeaths: 4762406,
    NewRecovered: 0,
    TotalRecovered: 0,
    Date: "2021-09-29T12:04:59.613Z"
    }
  const tree = renderer.create(<GlobalCard data={globalData} />).toJSON();
  const result = renderer.create(<GlobalCard data={globalData} />);
 
});
