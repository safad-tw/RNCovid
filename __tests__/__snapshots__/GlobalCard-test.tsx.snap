// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders snapshot as expected 1`] = `
<View
  style={
    Object {
      "borderRadius": 16,
      "marginVertical": 8,
    }
  }
>
  <RNSVGSvgView
    bbHeight={250}
    bbWidth={734}
    focusable={false}
    height={250}
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
          "borderWidth": 0,
        },
        Object {
          "flex": 0,
          "height": 250,
          "width": 734,
        },
      ]
    }
    width={734}
  >
    <RNSVGGroup>
      <RNSVGDefs>
        <RNSVGLinearGradient
          gradient={
            Array [
              0,
              -1051649,
              1,
              -1052689,
            ]
          }
          gradientTransform={null}
          gradientUnits={1}
          name="backgroundGradient"
          x1={0}
          x2={734}
          y1={250}
          y2={0}
        />
        <RNSVGLinearGradient
          gradient={
            Array [
              0,
              436207616,
              1,
              0,
            ]
          }
          gradientTransform={null}
          gradientUnits={1}
          name="fillShadowGradient"
          x1={0}
          x2={0}
          y1={0}
          y2={250}
        />
      </RNSVGDefs>
      <RNSVGDefs />
      <RNSVGRect
        fill={
          Array [
            1,
            "backgroundGradient",
          ]
        }
        height={250}
        propList={
          Array [
            "fill",
          ]
        }
        rx={16}
        ry={16}
        width="100%"
        x={0}
        y={0}
      />
      <RNSVGGroup>
        <RNSVGLine
          propList={
            Array [
              "stroke",
              "strokeWidth",
              "strokeDasharray",
            ]
          }
          stroke={855638016}
          strokeDasharray={
            Array [
              "5",
              "10",
            ]
          }
          strokeWidth={1}
          x1={0}
          x2={734}
          y1={16}
          y2={16}
        />
        <RNSVGLine
          propList={
            Array [
              "stroke",
              "strokeWidth",
              "strokeDasharray",
            ]
          }
          stroke={855638016}
          strokeDasharray={
            Array [
              "5",
              "10",
            ]
          }
          strokeWidth={1}
          x1={0}
          x2={734}
          y1={62.875}
          y2={62.875}
        />
        <RNSVGLine
          propList={
            Array [
              "stroke",
              "strokeWidth",
              "strokeDasharray",
            ]
          }
          stroke={855638016}
          strokeDasharray={
            Array [
              "5",
              "10",
            ]
          }
          strokeWidth={1}
          x1={0}
          x2={734}
          y1={109.75}
          y2={109.75}
        />
        <RNSVGLine
          propList={
            Array [
              "stroke",
              "strokeWidth",
              "strokeDasharray",
            ]
          }
          stroke={855638016}
          strokeDasharray={
            Array [
              "5",
              "10",
            ]
          }
          strokeWidth={1}
          x1={0}
          x2={734}
          y1={156.625}
          y2={156.625}
        />
        <RNSVGLine
          propList={
            Array [
              "stroke",
              "strokeWidth",
              "strokeDasharray",
            ]
          }
          stroke={855638016}
          strokeDasharray={
            Array [
              "5",
              "10",
            ]
          }
          strokeWidth={1}
          x1={0}
          x2={734}
          y1={203.5}
          y2={203.5}
        />
      </RNSVGGroup>
      <RNSVGGroup>
        <RNSVGText
          content={null}
          dx={Array []}
          dy={Array []}
          fill={3422552064}
          font={
            Object {
              "fontSize": 12,
              "textAnchor": "end",
            }
          }
          matrix={
            Array [
              1,
              0,
              -0,
              1,
              0,
              0,
            ]
          }
          propList={
            Array [
              "fill",
            ]
          }
          rotate={Array []}
          x={
            Array [
              52,
            ]
          }
          y={
            Array [
              203.5,
            ]
          }
        >
          <RNSVGTSpan
            content="0"
            dx={Array []}
            dy={Array []}
            font={Object {}}
            rotate={Array []}
            x={Array []}
            y={Array []}
          />
        </RNSVGText>
        <RNSVGText
          content={null}
          dx={Array []}
          dy={Array []}
          fill={3422552064}
          font={
            Object {
              "fontSize": 12,
              "textAnchor": "end",
            }
          }
          matrix={
            Array [
              1,
              0,
              -0,
              1,
              0,
              0,
            ]
          }
          propList={
            Array [
              "fill",
            ]
          }
          rotate={Array []}
          x={
            Array [
              52,
            ]
          }
          y={
            Array [
              156.625,
            ]
          }
        >
          <RNSVGTSpan
            content="58112"
            dx={Array []}
            dy={Array []}
            font={Object {}}
            rotate={Array []}
            x={Array []}
            y={Array []}
          />
        </RNSVGText>
        <RNSVGText
          content={null}
          dx={Array []}
          dy={Array []}
          fill={3422552064}
          font={
            Object {
              "fontSize": 12,
              "textAnchor": "end",
            }
          }
          matrix={
            Array [
              1,
              0,
              -0,
              1,
              0,
              0,
            ]
          }
          propList={
            Array [
              "fill",
            ]
          }
          rotate={Array []}
          x={
            Array [
              52,
            ]
          }
          y={
            Array [
              109.75,
            ]
          }
        >
          <RNSVGTSpan
            content="116225"
            dx={Array []}
            dy={Array []}
            font={Object {}}
            rotate={Array []}
            x={Array []}
            y={Array []}
          />
        </RNSVGText>
        <RNSVGText
          content={null}
          dx={Array []}
          dy={Array []}
          fill={3422552064}
          font={
            Object {
              "fontSize": 12,
              "textAnchor": "end",
            }
          }
          matrix={
            Array [
              1,
              0,
              -0,
              1,
              0,
              0,
            ]
          }
          propList={
            Array [
              "fill",
            ]
          }
          rotate={Array []}
          x={
            Array [
              52,
            ]
          }
          y={
            Array [
              62.875,
            ]
          }
        >
          <RNSVGTSpan
            content="174337"
            dx={Array []}
            dy={Array []}
            font={Object {}}
            rotate={Array []}
            x={Array []}
            y={Array []}
          />
        </RNSVGText>
        <RNSVGText
          content={null}
          dx={Array []}
          dy={Array []}
          fill={3422552064}
          font={
            Object {
              "fontSize": 12,
              "textAnchor": "end",
            }
          }
          matrix={
            Array [
              1,
              0,
              -0,
              1,
              0,
              0,
            ]
          }
          propList={
            Array [
              "fill",
            ]
          }
          rotate={Array []}
          x={
            Array [
              52,
            ]
          }
          y={
            Array [
              16,
            ]
          }
        >
          <RNSVGTSpan
            content="232449"
            dx={Array []}
            dy={Array []}
            font={Object {}}
            rotate={Array []}
            x={Array []}
            y={Array []}
          />
        </RNSVGText>
      </RNSVGGroup>
      <RNSVGGroup>
        <RNSVGText
          content={null}
          dx={Array []}
          dy={Array []}
          fill={3422552064}
          font={
            Object {
              "fontSize": 12,
              "textAnchor": "middle",
            }
          }
          matrix={
            Array [
              1,
              0,
              -0,
              1,
              0,
              0,
            ]
          }
          propList={
            Array [
              "fill",
            ]
          }
          rotate={Array []}
          x={
            Array [
              96,
            ]
          }
          y={
            Array [
              227.5,
            ]
          }
        >
          <RNSVGTSpan
            content="Global deaths(K)"
            dx={Array []}
            dy={Array []}
            font={Object {}}
            rotate={Array []}
            x={Array []}
            y={Array []}
          />
        </RNSVGText>
        <RNSVGText
          content={null}
          dx={Array []}
          dy={Array []}
          fill={3422552064}
          font={
            Object {
              "fontSize": 12,
              "textAnchor": "middle",
            }
          }
          matrix={
            Array [
              1,
              0,
              -0,
              1,
              0,
              0,
            ]
          }
          propList={
            Array [
              "fill",
            ]
          }
          rotate={Array []}
          x={
            Array [
              319.33333333333337,
            ]
          }
          y={
            Array [
              227.5,
            ]
          }
        >
          <RNSVGTSpan
            content="Recoveries(K)"
            dx={Array []}
            dy={Array []}
            font={Object {}}
            rotate={Array []}
            x={Array []}
            y={Array []}
          />
        </RNSVGText>
        <RNSVGText
          content={null}
          dx={Array []}
          dy={Array []}
          fill={3422552064}
          font={
            Object {
              "fontSize": 12,
              "textAnchor": "middle",
            }
          }
          matrix={
            Array [
              1,
              0,
              -0,
              1,
              0,
              0,
            ]
          }
          propList={
            Array [
              "fill",
            ]
          }
          rotate={Array []}
          x={
            Array [
              542.6666666666667,
            ]
          }
          y={
            Array [
              227.5,
            ]
          }
        >
          <RNSVGTSpan
            content="Active Cases(K)"
            dx={Array []}
            dy={Array []}
            font={Object {}}
            rotate={Array []}
            x={Array []}
            y={Array []}
          />
        </RNSVGText>
      </RNSVGGroup>
      <RNSVGGroup>
        <RNSVGRect
          fill={
            Array [
              1,
              "fillShadowGradient",
            ]
          }
          height={3.8414909808577837}
          propList={
            Array [
              "fill",
            ]
          }
          rx={0}
          width={32}
          x={80}
          y={199.65850901914223}
        />
        <RNSVGRect
          fill={
            Array [
              1,
              "fillShadowGradient",
            ]
          }
          height={0}
          propList={
            Array [
              "fill",
            ]
          }
          rx={0}
          width={32}
          x={303.33333333333337}
          y={203.5}
        />
        <RNSVGRect
          fill={
            Array [
              1,
              "fillShadowGradient",
            ]
          }
          height={187.5}
          propList={
            Array [
              "fill",
            ]
          }
          rx={0}
          width={32}
          x={526.6666666666667}
          y={16}
        />
      </RNSVGGroup>
      <RNSVGGroup />
      <RNSVGGroup>
        <RNSVGRect
          fill={2566914048}
          height={2}
          propList={
            Array [
              "fill",
            ]
          }
          width={32}
          x={80}
          y={199.65850901914223}
        />
        <RNSVGRect
          fill={2566914048}
          height={2}
          propList={
            Array [
              "fill",
            ]
          }
          width={32}
          x={303.33333333333337}
          y={203.5}
        />
        <RNSVGRect
          fill={2566914048}
          height={2}
          propList={
            Array [
              "fill",
            ]
          }
          width={32}
          x={526.6666666666667}
          y={16}
        />
      </RNSVGGroup>
    </RNSVGGroup>
  </RNSVGSvgView>
</View>
`;

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "borderRadius": 16,
      "marginVertical": 8,
    }
  }
>
  <RNSVGSvgView
    bbHeight={250}
    bbWidth={734}
    focusable={false}
    height={250}
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
          "borderWidth": 0,
        },
        Object {
          "flex": 0,
          "height": 250,
          "width": 734,
        },
      ]
    }
    width={734}
  >
    <RNSVGGroup>
      <RNSVGDefs>
        <RNSVGLinearGradient
          gradient={
            Array [
              0,
              -1051649,
              1,
              -1052689,
            ]
          }
          gradientTransform={null}
          gradientUnits={1}
          name="backgroundGradient"
          x1={0}
          x2={734}
          y1={250}
          y2={0}
        />
        <RNSVGLinearGradient
          gradient={
            Array [
              0,
              436207616,
              1,
              0,
            ]
          }
          gradientTransform={null}
          gradientUnits={1}
          name="fillShadowGradient"
          x1={0}
          x2={0}
          y1={0}
          y2={250}
        />
      </RNSVGDefs>
      <RNSVGDefs />
      <RNSVGRect
        fill={
          Array [
            1,
            "backgroundGradient",
          ]
        }
        height={250}
        propList={
          Array [
            "fill",
          ]
        }
        rx={16}
        ry={16}
        width="100%"
        x={0}
        y={0}
      />
      <RNSVGGroup>
        <RNSVGLine
          propList={
            Array [
              "stroke",
              "strokeWidth",
              "strokeDasharray",
            ]
          }
          stroke={855638016}
          strokeDasharray={
            Array [
              "5",
              "10",
            ]
          }
          strokeWidth={1}
          x1={0}
          x2={734}
          y1={16}
          y2={16}
        />
        <RNSVGLine
          propList={
            Array [
              "stroke",
              "strokeWidth",
              "strokeDasharray",
            ]
          }
          stroke={855638016}
          strokeDasharray={
            Array [
              "5",
              "10",
            ]
          }
          strokeWidth={1}
          x1={0}
          x2={734}
          y1={62.875}
          y2={62.875}
        />
        <RNSVGLine
          propList={
            Array [
              "stroke",
              "strokeWidth",
              "strokeDasharray",
            ]
          }
          stroke={855638016}
          strokeDasharray={
            Array [
              "5",
              "10",
            ]
          }
          strokeWidth={1}
          x1={0}
          x2={734}
          y1={109.75}
          y2={109.75}
        />
        <RNSVGLine
          propList={
            Array [
              "stroke",
              "strokeWidth",
              "strokeDasharray",
            ]
          }
          stroke={855638016}
          strokeDasharray={
            Array [
              "5",
              "10",
            ]
          }
          strokeWidth={1}
          x1={0}
          x2={734}
          y1={156.625}
          y2={156.625}
        />
        <RNSVGLine
          propList={
            Array [
              "stroke",
              "strokeWidth",
              "strokeDasharray",
            ]
          }
          stroke={855638016}
          strokeDasharray={
            Array [
              "5",
              "10",
            ]
          }
          strokeWidth={1}
          x1={0}
          x2={734}
          y1={203.5}
          y2={203.5}
        />
      </RNSVGGroup>
      <RNSVGGroup>
        <RNSVGText
          content={null}
          dx={Array []}
          dy={Array []}
          fill={3422552064}
          font={
            Object {
              "fontSize": 12,
              "textAnchor": "end",
            }
          }
          matrix={
            Array [
              1,
              0,
              -0,
              1,
              0,
              0,
            ]
          }
          propList={
            Array [
              "fill",
            ]
          }
          rotate={Array []}
          x={
            Array [
              52,
            ]
          }
          y={
            Array [
              203.5,
            ]
          }
        >
          <RNSVGTSpan
            content="0"
            dx={Array []}
            dy={Array []}
            font={Object {}}
            rotate={Array []}
            x={Array []}
            y={Array []}
          />
        </RNSVGText>
        <RNSVGText
          content={null}
          dx={Array []}
          dy={Array []}
          fill={3422552064}
          font={
            Object {
              "fontSize": 12,
              "textAnchor": "end",
            }
          }
          matrix={
            Array [
              1,
              0,
              -0,
              1,
              0,
              0,
            ]
          }
          propList={
            Array [
              "fill",
            ]
          }
          rotate={Array []}
          x={
            Array [
              52,
            ]
          }
          y={
            Array [
              156.625,
            ]
          }
        >
          <RNSVGTSpan
            content="58112"
            dx={Array []}
            dy={Array []}
            font={Object {}}
            rotate={Array []}
            x={Array []}
            y={Array []}
          />
        </RNSVGText>
        <RNSVGText
          content={null}
          dx={Array []}
          dy={Array []}
          fill={3422552064}
          font={
            Object {
              "fontSize": 12,
              "textAnchor": "end",
            }
          }
          matrix={
            Array [
              1,
              0,
              -0,
              1,
              0,
              0,
            ]
          }
          propList={
            Array [
              "fill",
            ]
          }
          rotate={Array []}
          x={
            Array [
              52,
            ]
          }
          y={
            Array [
              109.75,
            ]
          }
        >
          <RNSVGTSpan
            content="116225"
            dx={Array []}
            dy={Array []}
            font={Object {}}
            rotate={Array []}
            x={Array []}
            y={Array []}
          />
        </RNSVGText>
        <RNSVGText
          content={null}
          dx={Array []}
          dy={Array []}
          fill={3422552064}
          font={
            Object {
              "fontSize": 12,
              "textAnchor": "end",
            }
          }
          matrix={
            Array [
              1,
              0,
              -0,
              1,
              0,
              0,
            ]
          }
          propList={
            Array [
              "fill",
            ]
          }
          rotate={Array []}
          x={
            Array [
              52,
            ]
          }
          y={
            Array [
              62.875,
            ]
          }
        >
          <RNSVGTSpan
            content="174337"
            dx={Array []}
            dy={Array []}
            font={Object {}}
            rotate={Array []}
            x={Array []}
            y={Array []}
          />
        </RNSVGText>
        <RNSVGText
          content={null}
          dx={Array []}
          dy={Array []}
          fill={3422552064}
          font={
            Object {
              "fontSize": 12,
              "textAnchor": "end",
            }
          }
          matrix={
            Array [
              1,
              0,
              -0,
              1,
              0,
              0,
            ]
          }
          propList={
            Array [
              "fill",
            ]
          }
          rotate={Array []}
          x={
            Array [
              52,
            ]
          }
          y={
            Array [
              16,
            ]
          }
        >
          <RNSVGTSpan
            content="232449"
            dx={Array []}
            dy={Array []}
            font={Object {}}
            rotate={Array []}
            x={Array []}
            y={Array []}
          />
        </RNSVGText>
      </RNSVGGroup>
      <RNSVGGroup>
        <RNSVGText
          content={null}
          dx={Array []}
          dy={Array []}
          fill={3422552064}
          font={
            Object {
              "fontSize": 12,
              "textAnchor": "middle",
            }
          }
          matrix={
            Array [
              1,
              0,
              -0,
              1,
              0,
              0,
            ]
          }
          propList={
            Array [
              "fill",
            ]
          }
          rotate={Array []}
          x={
            Array [
              96,
            ]
          }
          y={
            Array [
              227.5,
            ]
          }
        >
          <RNSVGTSpan
            content="Global deaths(K)"
            dx={Array []}
            dy={Array []}
            font={Object {}}
            rotate={Array []}
            x={Array []}
            y={Array []}
          />
        </RNSVGText>
        <RNSVGText
          content={null}
          dx={Array []}
          dy={Array []}
          fill={3422552064}
          font={
            Object {
              "fontSize": 12,
              "textAnchor": "middle",
            }
          }
          matrix={
            Array [
              1,
              0,
              -0,
              1,
              0,
              0,
            ]
          }
          propList={
            Array [
              "fill",
            ]
          }
          rotate={Array []}
          x={
            Array [
              319.33333333333337,
            ]
          }
          y={
            Array [
              227.5,
            ]
          }
        >
          <RNSVGTSpan
            content="Recoveries(K)"
            dx={Array []}
            dy={Array []}
            font={Object {}}
            rotate={Array []}
            x={Array []}
            y={Array []}
          />
        </RNSVGText>
        <RNSVGText
          content={null}
          dx={Array []}
          dy={Array []}
          fill={3422552064}
          font={
            Object {
              "fontSize": 12,
              "textAnchor": "middle",
            }
          }
          matrix={
            Array [
              1,
              0,
              -0,
              1,
              0,
              0,
            ]
          }
          propList={
            Array [
              "fill",
            ]
          }
          rotate={Array []}
          x={
            Array [
              542.6666666666667,
            ]
          }
          y={
            Array [
              227.5,
            ]
          }
        >
          <RNSVGTSpan
            content="Active Cases(K)"
            dx={Array []}
            dy={Array []}
            font={Object {}}
            rotate={Array []}
            x={Array []}
            y={Array []}
          />
        </RNSVGText>
      </RNSVGGroup>
      <RNSVGGroup>
        <RNSVGRect
          fill={
            Array [
              1,
              "fillShadowGradient",
            ]
          }
          height={3.8414909808577837}
          propList={
            Array [
              "fill",
            ]
          }
          rx={0}
          width={32}
          x={80}
          y={199.65850901914223}
        />
        <RNSVGRect
          fill={
            Array [
              1,
              "fillShadowGradient",
            ]
          }
          height={0}
          propList={
            Array [
              "fill",
            ]
          }
          rx={0}
          width={32}
          x={303.33333333333337}
          y={203.5}
        />
        <RNSVGRect
          fill={
            Array [
              1,
              "fillShadowGradient",
            ]
          }
          height={187.5}
          propList={
            Array [
              "fill",
            ]
          }
          rx={0}
          width={32}
          x={526.6666666666667}
          y={16}
        />
      </RNSVGGroup>
      <RNSVGGroup />
      <RNSVGGroup>
        <RNSVGRect
          fill={2566914048}
          height={2}
          propList={
            Array [
              "fill",
            ]
          }
          width={32}
          x={80}
          y={199.65850901914223}
        />
        <RNSVGRect
          fill={2566914048}
          height={2}
          propList={
            Array [
              "fill",
            ]
          }
          width={32}
          x={303.33333333333337}
          y={203.5}
        />
        <RNSVGRect
          fill={2566914048}
          height={2}
          propList={
            Array [
              "fill",
            ]
          }
          width={32}
          x={526.6666666666667}
          y={16}
        />
      </RNSVGGroup>
    </RNSVGGroup>
  </RNSVGSvgView>
</View>
`;
